.github/workflows/design-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: Design Changes Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      description:
+        description: 'What are we updating (eg. update button destructive color)?'
+        default: ''
+        required: true
+      jira:
+        description: 'What is the related JIRA (eg. SW-1234)?'
+        default: ''
+        required: true
+      email:
+        description: 'Your terraformation email'
+        default: ''
+        required: true
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Create branch
+        run: git checkout -b `echo design-workflow-${{ github.actor }}-${{ github.event.inputs.jira }}-\`date +"%y-%m-%d"\`  | sed "s/ /-/g"`
+
+      - name: Install Dependencies
+        run: yarn install --network-timeout 100000
+
+      - name: Build dictionary 
+        run: yarn build-dictionary
+
+      - name: Bump package patch version
+        run: yarn bump patch
+
+      - name: Config user
+        run: |
+          git config --global user.email ${{ github.event.inputs.email }}
```

Comment:
I'll be curious to see what this does if a user has more than one email address on their GitHub account.

Reply:
I have 2 emails with my github account (personal email and terraformation email), let me try both emails.

Reply:
Both emails were accepted, not sure how this manifests in the PR creation - I see my username in the PRs.